qutip/floquet.py

```diff
@@ -702,20 +699,14 @@ def floquet_master_equation_tensor(Alist, f_energies):
         for J in range(N * N):
             c, d = vec2mat_index(N, J)
 
-            R = -1.0j * (f_energies[a] - f_energies[b])*(a == c)*(b == d)
-            Rdata_lil[I, J] = R
-
-            for A in Alist:
-                s1 = s2 = 0
-                for n in range(N):
-                    s1 += A[a, n] * (n == c) * (n == d) - A[n, a] * \
-                        (a == c) * (a == d)
-                    s2 += (A[n, a] + A[n, b]) * (a == c) * (b == d)
-
-                dR = (a == b) * s1 - 0.5 * (1 - (a == b)) * s2
+            if a == b and c == d:
+                Rdata_lil[I, J] += sum(A[a, c] for A in Alist)
 
-                if dR != 0.0:
-                    Rdata_lil[I, J] += dR
+            if a == c and b == d:
+                Rdata_lil[I, J] += -0.5 * sum(
+                    sum(A[n, a] + A[n, b] for n in range(N))
+                    for A in Alist
+                )
```

Comment:
Reworked the way computation is done. Using my simple code from #772, tensor computation times drops from around 50 seconds to less than a second.

Output was the same as the previous code, except from the `f_energies` part.